src/app/config/projectsJsonData.ts

```diff
@@ -59,6 +59,21 @@ export default class projectsJsonData {
     return hosts;
   }
 
+  getSupportedScripts(projectType: string) {
```

Comment:
"Scripts" is confusing.  I see that this is for the "scriptType" questions and while I'm not a fan of that labeling either, I think this should match.